src/Components/Nav/Nav.js

```diff
@@ -1,7 +1,84 @@
 import React, { Component } from "react";
+import { Link } from "react-router-dom";
+import Navfirst from "./Navfirst";
+import Navsecond from "./Navsecond";
+import { HEAD } from "./NavData";
+import "./Nav.scss";
 
 export default class Nav extends Component {
+  constructor() {
+    super();
+    this.state = {
+      currentId: 1,
+    };
+  }
+
+  clickHandler = (id) => {
+    this.setState({ currentId: id });
+  };
+
   render() {
-    return <div></div>;
+    return (
+      <div>
+        <nav className="navbar">
+          <div className="nav_cm">
```

Comment:
className을 굳이 nav_cm이라고 하신 이유가 있으실까요?
sass를 통해서 최상단 태그로 nesting을 하기 때문에 굳이 부모태그의 classname을 작성하실 필요가 없을 것 같습니다. 
cm도 직관적으로 아래와 같이 작성해주시면 더욱 가독성에 좋겠네요!
```suggestion
          <div className="comment">
```

Reply:
1. nav_cm -> nav_wrap 으로 수정했습니다 !